index.html

```diff
@@ -78,97 +78,15 @@ <h1 class="heading1">
-                        <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
-                        <li><div class="button2"><button2>CSS</button2><div><li>  
-                        <li><div class="button2"><button2>Javascript</button2><div><li>
-                        <li><div class="button2"><button2>HTML</button2><div><li>
-                      </ul>
-                      <button class="button">See Project</button>
-                  </div>
-                </div>
-              </div>
-
-              <div class="griditem">
-                <div class="placeholder"></div>
-                  <div class=whitespace>
-                    <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
-                        <ul class="minimenu">
-                          <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
-                          <li><div class="button2"><button2>CSS</button2><div><li>  
-                          <li><div class="button2"><button2>Javascript</button2><div><li>
-                          <li><div class="button2"><button2>HTML</button2><div><li>
-                        </ul>
-                        <button class="button">See Project</button>
-                    </div>
-                  </div>
-                </div>
-
-                <div class="griditem">
-                  <div class="placeholder"></div>
-                    <div class=whitespace>
-                      <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
-                          <ul class="minimenu">
-                            <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
-                            <li><div class="button2"><button2>CSS</button2><div><li>  
-                            <li><div class="button2"><button2>Javascript</button2><div><li>
-                            <li><div class="button2"><button2>HTML</button2><div><li>
-                          </ul>
-                          <button class="button">See Project</button>
-                      </div>
-                    </div>
-                  </div>
-
-                  <div class="griditem">
-                    <div class="placeholder"></div>
-                      <div class=whitespace>
-                        <div><h3>Multi-Post Stories<br> Gain+Glory</h3>
-                            <ul class="minimenu">
-                              <li><div class="button2"><button2>Ruby on Rails</button2><div><li>
-                              <li><div class="button2"><button2>CSS</button2><div><li>  
-                              <li><div class="button2"><button2>Javascript</button2><div><li>
-                              <li><div class="button2"><button2>HTML</button2><div><li>
-                            </ul>
-                            <button class="button">See Project</button>
-                        </div>
-                      </div>
-                    </div>
+      <div id="projects-container" class=grid-container>
```

Comment:
Kindly consider creating new elements through javascript 
 as HTML for the project section needs to be loaded through Javascript file.

Reply:
Ok Kaiser, thanks.